spec/services/shopify_voider_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+require_relative '../../lib/active_merchant/billing/gateways/shopify.rb'
+
+describe ShopifyVoider do
```

Comment:
Could we establish a namespace module to put things like this in? `Spree::Retail` maybe?